validphys2/src/validphys/pseudodata.py

```diff
@@ -296,3 +296,8 @@ def recreate_pdf_pseudodata(_recreate_pdf_pseudodata, pdfreplicas, pdf_tr_masks)
     :py:func:`validphys.pseudodata.recreate_fit_pseudodata`
     """
     return recreate_fit_pseudodata(_recreate_pdf_pseudodata, pdfreplicas, pdf_tr_masks)
+
+pdf_tr_masks_no_table = collect('replica_training_mask', ('pdfreplicas', 'fitenvironment'))
+def recreate_pdf_pseudodata_no_table(_recreate_pdf_pseudodata, pdfreplicas, pdf_tr_masks_no_table):
+    return recreate_fit_pseudodata(_recreate_pdf_pseudodata, pdfreplicas, pdf_tr_masks_no_table)
```

Comment:
```suggestion
recreate_pdf_pseudodata_no_table = recreate_fit_pseudodata
```

why would this not be exctly the same?

Reply:
For python it is, for vp I wouldn't be surprised if it's not (but didn't check)

Reply:
When you asked the question you were probably thinking about the decorator? I did as well, that's why I accepted it at first, but the reason it has to be done this way is that the inputs are different (and also the input is what creates the table in one of the cases)

Reply:
But I am confused, the inputs look exactly the same to me?

(but yes, I thought you did it because of the decorator and that's why I thought it would be the same)

Reply:
ahh! Okokok. Got it now, sorry.